source/includes/_changes.md

```diff
@@ -0,0 +1,144 @@
+# Dataset Changes
+
+The changes of a dataset describe subsequent changes of states that affected the different sections of a dataset. Every action taken on any resource through POST, PUT or DELETE creates a change object that can be retrieved and acted upon.
+
+## The change object
+
+> An example change object
+
+```json
+{
+    "change_uid": 125,
+    "dataset": {
+        "dataset_uid": "dataset",
+        "domain": "domain"
+    },
+    "user": {
+        "username": "john_smith"
+    },
+    "timestamp": "2017-06-07T15:16:05.701266+00:00",
+    "diff": {
+        "metadata": [{
+            "path": ["default", "description"],
+            "old_value": null,
+            "new_value": "dataset title",
+            "operation_type": "create"
+        }]
+    },
+    "sections": [
+        "metadata"
+    ]
+}
+```
+
+The change object contains:
+
+* a unique identifier
+* a reference to the dataset that changed
+* a reference to the user who created the new state
+* the day and time at which the change occured
+* the difference between the old and the new state of the dataset
+* the sections in which the change happened
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`change_uid` <br> *string* | unique identifier for the change
```

Comment:
Also to ensure homogeneity in the documentation, could you please write the first letter of the first word of each description in capital letter?